### ISSUE TITLE
UX: Content is not visible on smaller screens when too long input

### ISSUE THREAD

**Issue by alina-lapina** [statisticsnorway/klass-subsets-client#192]
**Describe the bug**
Content is not visible on smaller screens when too long input

**App version**
Since responsive design implemented v0.6.0

**To Reproduce**
1.  Choose a small screen (narrow browser window)
2.  Type a really long name to make the brief expand out of the page. 
3.  Scroll bar appears, but cannot show the left or right side of the form

**Expected behavior**
All content has to be visible on screen on any screen size.

**Screenshots**
![image](https://user-images.githubusercontent.com/21969896/103164437-72a22080-480b-11eb-880d-b7350a935257.png)
